src/app.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="%svelte.i18.locale%">
```

Comment:
having the lang attribute rendered server-side is an important milestone for i18n and SEO